### ISSUE TITLE
The comment might be misleading when size limit fails

### ISSUE THREAD

**Issue by ad1992** [ai/size-limit#332]
Below is the result json after running size-limit

```
[
  {
    "name": "dist/excalidraw.production.min.js",
    "passed": false,
    "size": 286003,
    "sizeLimit": 285000,
    "running": 0.06112,
    "loading": 5.58599609375
  },
  {
    "name": "dist/excalidraw-assets/locales",
    "passed": true,
    "size": 267745,
    "sizeLimit": 270000,
    "running": 0.017186666666666666,
    "loading": 5.[229](https://github.com/excalidraw/excalidraw/actions/runs/5659242959/job/15332367714?pr=6803#step:6:230)39453125
  },
  {
    "name": "dist/excalidraw-assets/vendor*.js",
    "passed": true,
    "size": 28735,
    "sizeLimit": 30000,
    "running": 0.021683333333333336,
    "loading": 0.561[230](https://github.com/excalidraw/excalidraw/actions/runs/5659242959/job/15332367714?pr=6803#step:6:231)46875
  }
]
```
As you can see the size is exceeded for `excalidraw.production.min.js`

However in the comment posted by size-limit, this is what is shown

<img width="827" alt="Screenshot 2023-07-26 at 8 45 20 AM" src="https://github.com/ai/size-limit/assets/11256141/a1f9c98a-26b7-465e-afb0-1d05204f3f91">

And its not clear as the size `279.33 kB` is within the limit of `excalidraw.production.min.js` (285 kB)

**Comment by ai** [comment]
Seems like you need to open an issue in [size-limit-action](https://github.com/andresz1/size-limit-action) since that project has a different maintainer

**Comment by ad1992** [comment]
@ai before I make the change for this fix in the other gh action, I would like to inform that the difference is as the gh action library uses `bytes` (KIB) as mentioned [here](https://github.com/andresz1/size-limit-action/issues/106#issuecomment-1709593070) so should I change it in the gh action so it considers in KB or change here so it considers in KIB ?

**Comment by ai** [comment]
Yes, it is better to keep the same size format for both tool

**Comment by ad1992** [comment]
> Yes, it is better to keep the same size format for both tool

yep so as I asked above, which repo would be better to make the change? This repo or Gh actions repo?

**Comment by ai** [comment]
Gh actions repo
